assets-and-images.md

```diff
@@ -4,10 +4,12 @@ title: Assets and Images
 permalink: /assets-and-images/
 ---
 
-Flutter applications include both code and assets (sometimes called resources).
-Assets can include static data, configuration files, or anything else an
-application needs to function; in particular, assets include icons and other
-images displayed in the UI.
+Flutter applications can include both code and _assets_
+(sometimes called resources). An asset is a file that is bundled
+and deployed with your application, and is accessible at runtime.
+Common types of assets include static data
+(for example, JSON files), configuration files, icons,
```

Comment:
Yes, although this document doesn't talk much about how audio is handled... Do we have a doc for audio?